<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RequestsAdminWidget</class>
    <widget class="QWidget" name="RequestsAdminWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QTableView" name="tableView">
                    <property name="selectionMode">
                        <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                        <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="approveButton">
                            <property name="text">
                                <string>Approve</string>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: #4CAF50; color: white;</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="rejectButton">
                            <property name="text">
                                <string>Reject</string>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: #f44336; color: white;</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="refreshButton">
                            <property name="text">
                                <string>Refresh</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
